utility/importer/sources.gni

```diff
@@ -30,6 +30,8 @@ if (!is_android) {
     "//chrome/app:branded_strings",
     "//components/os_crypt/sync",
     "//components/password_manager/core/browser",
+    "//components/user_data_importer/common",
+    "//components/user_data_importer/common",
```

Comment:
this target is duplicate.

Reply:
Good catch, fixed here https://github.com/brave/brave-core/pull/28944/commits/7822cd14309a17a5eb30103ea02533f436ce389c